More C++ examples. CUDA and MPI+CUDA. Draft 0, review required (#793)
NREL/HPC: docs/Documentation/Applications/chemicalKinetics.md

$ make
``` 

> ###Chemkin

Reviewer: ```suggestion
### Chemkin
```

> Chemkin GUI can be accessed though a [FastX virtual desktop](https://nrel.github.io/HPC/Documentation/Viz_Analytics/virtualgl_fastx/). To use Chemkin Reaction Workbench in a FastX session, open a terminal within the FastX desktop and execute the following:

Reviewer: ```suggestion
The Chemkin GUI can be accessed though a [FastX virtual desktop](../Viz_Analytics/virtualgl_fastx.md). To use Chemkin Reaction Workbench in a FastX session, open a terminal within the FastX desktop and execute the following:
```

```

> $ module load ansys/2025R1

Reviewer: Does this have to be this Ansys version, or could you change to "module load ansys" so that we don't have to remember to update versions later?

$ run_rdworkbench.sh
```

> For resourse intensive operations, please run Chemkin through a dedicated compute node. The steps are as follows:

Reviewer: ```suggestion
For resource intensive operations, please run Chemkin through a dedicated compute node. The steps are as follows:
```

> 1. Open a terminal in a FastX session and ask for an [allocation](https://nrel.github.io/HPC/Documentation/Slurm/interactive_jobs/). For example,

Reviewer: ```suggestion
1. Open a terminal in a FastX session and ask for an [allocation](../Slurm/interactive_jobs.md). For example,
```

```

> $ salloc -A <projectname> -t 02:00:00 --nodes=1 --ntasks-per-node=20 --gres=gpu:1

Reviewer: Do you need to request any additional memory?

```
2. Wait until you obtain an allocation, The terminal will display `<username>@<nodename>` when successful.
3. Open a new terminal tab by right clicking on empty in the terminal. In the new terminal tab, execute the following to connect to the node you have been allocated.
```
$ ssh -X <nodename>
```

> 4. You are now on a compute node with [X forwarding](https://en.wikipedia.org/wiki/X_Window_System) to FastX desktop session, ready to run GUI applications. To run Chemkin, execute the following in this new terminal tab:

Reviewer: ```suggestion
4. You are now on a compute node with [X forwarding](https://en.wikipedia.org/wiki/X_Window_System) to a FastX desktop session, ready to run GUI applications. To run Chemkin, execute the following in this new terminal tab:
```
